entrypoint.sh

```diff
@@ -33,6 +33,11 @@ case $CLOUD in
     ;;
 esac
 
```

Comment:
I dont think this change has anything to do with the stated issue, could it be removed from the PR